Bigtable/src/Instance.php

```diff
@@ -0,0 +1,341 @@
+     *     @type array $clusters [] {
+     *           array {
+     *                 string $clusterId
+     *                 string $locationId
+     *                 int $serveNodes
+     *                 int $storageType The storage media type for persisting Bigtable data. Possible values include `Instance::STORAGE_TYPE_SSD` and `Instance::STORAGE_TYPE_HDD`.
+     *                 **Defaults to** `Instance::STORAGE_TYPE_UNSPECIFIED`.
+     *          }
+     * }
+     * @return LongRunningOperation<Instance>
+     * @codingStandardsIgnoreEnd
+     */
+    public function create(array $options = [])
+    {
+        $projectName = InstanceAdminClient::projectName($this->projectId);
+        $instanceId = InstanceAdminClient::parseName($this->name)['instance'];
+        $displayName = isset($options['displayName']) ? $options['displayName'] : $instanceId;
+        $type = isset($options['instanceType']) ? $options['instanceType'] : self::INSTANCE_TYPE_UNSPECIFIED;
+        $labels = isset($options['labels']) ? $options['labels'] : [];
+        $clusters = isset($options['clusters']) ? $options['clusters'] : [];
+
+        $operation = $this->connection->createInstance([
+            'parent' => $projectName,
+            'instanceId' => $instanceId,
+            'instance' => [
+                'displayName' => $displayName,
+                'type' => $type,
+                'labels' => $labels
+            ],
+            'clusters' => $this->clustersArray($clusters)
+        ]);
+        return $this->resumeOperation($operation['name'], $operation);
+    }
+
+    /**
+     * @param array $args [] {
+     *      array {
+     *             string $clusterId
+     *             string $locationId
+     *             int $serveNodes
+     *             int $storageType **Defaults to** `StorageType::STORAGE_TYPE_UNSPECIFIED`.
+     *      }
+     * }
+     * @return array
+    */
+    private function clustersArray($args)
+    {
+        $clusters = [];
+        foreach ($args as $value) {
+            $id = $value['clusterId'];
+            $clusters[$id] = [
+                'location' => InstanceAdminClient::locationName($this->projectId, $value['locationId']),
+                'serveNodes' => isset($value['serveNodes']) ? $value['serveNodes'] : '',
+                'defaultStorageType' => isset($value['storageType'])?$value['storageType']
+                :self::STORAGE_TYPE_UNSPECIFIED
+            ];
+        }
+        return $clusters;
+    }
+
+    /**
+     * Convert the simple instance name to a fully qualified name.
+     *
+     * @param string $instanceId The instance ID.
+     * @param string $projectId The project ID.
+     * @return string
+     */
+    private function fullyQualifiedInstanceName($instanceId, $projectId)
+    {
+        try {
```

Comment:
I think we can remove this try/catch block and the `ValidationException` import. @jdpedrie could we please make a note to update the instance class in spanner as well?